design-proposals/new-proposals-framework.md

```diff
@@ -0,0 +1,75 @@
+
+
+With this proposal, we aim to:
+
+- Optimize contributors' and reviewers' bandwidth and experience.
+- Strengthen SIG ownership and accountability.
+- Improve the alignment of enhancements with the project's goals and architecture.
+- Separate the enhancement process from community matters.
+
+# Goals
+
+1. Introduce a new repository, `kubevirt/enhancements`, to house all VEPs.
+2. Prioritize VEPs at the start of each development cycle to streamline review and focus efforts.
+3. Establish SIG-specific review processes with a dedicated reviewer for each VEP.
+4. Create issues to track the progress, maturity, and associated bugs for each accepted VEP.
+5. Establish ownership of the VEP and any additional SIG collaboration.
+
+## Non-Goals
+
+- Removing the central approval mechanism at this stage.
+- Full decentralization of VEP review and approval processes.
+
+## Repos
+
+- **Impacted Repository**: `kubevirt/enhancements` and kubevirt/community
+
+# Design
+
+The process includes the following key components:
+1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. [Design proposal template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md)
+2. **SIG Review and Collaboration**: Each VEP will have a target SIG, and the SIG will assign a dedicated reviewer to oversee the proposal, collaborate with other SIGs as needed, and provide feedback or veto when necessary.
+3. **Centralized Prioritization**: At the start of each release cycle, a centralized mechanism will prioritize accepted VEPs, focusing community efforts on these.
+4. **Visibility and Tracking**: The Author of an accepted VEPs will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback
+5. *Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement
+Includes all the relevant information, including the design and the state.
+The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
+
+## Scalability
+
+This design allows scalability by:
+- Distributing review responsibility across SIGs while maintaining central oversight.
+- Providing a framework for gradual decentralization as SIGs mature.
+- Centralized tracking to set priorities and ensure alignment across the project.
+- Uniform process 
+
+# Approvers
+The following individuals are proposed as the initial approvers for the kubevirt/enhancements repository. These approvers will be responsible for ensuring VEPs meet the required standards, align with the project’s goals and best practices.
+ - Luboslav Pivarc @xpivarc
+ - Vladik Romanovsky @vladikr
+
+# Implementation Phases
+
+1. **Preparation (Pre-v1.5)**:
```

Comment:
The dev cycle for `v1.5.0` is already underway so just merge this with the second bullet?